### PR TITLE
updates to map figure make

### PR DESCRIPTION
**Pull request by jordansread**

### CODE REVIEW

**Comment by lindsayplatt** [comment]
@jread-usgs is this ready to go? I just merged a PR, so you'll need to pull upstream before merging this

**Comment by jordansread** [comment]
@lindsaycarr can you review and merge?

File: R/wqp_mapping_functions.R [hunk 1]
```diff
@@ -81,3 +81,23 @@ plot_huc_sites <- function(hucs, sites, map.config, figure.name){
   }
   dev.off()
 }
+
+as.sites <- function(target_name){
```
Comment (lindsayplatt) [inline R/wqp_mapping_functions.R]: where does this get used? Can't find that

Comment (jordansread) [inline R/wqp_mapping_functions.R]: whoops. Other project code that snuck in there....

Comment (jordansread) [inline R/wqp_mapping_functions.R]: fixed now

**Pull request merged**